pkg/framework/util.go

```diff
@@ -31,9 +31,12 @@ import (
 )
 
 const (
-	defaultUIDPrefix       string = "e2e-cri-uid"
-	defaultNamespacePrefix string = "e2e-cri-namespace"
-	defaultAttempt         uint32 = 2
+	defaultUIDPrefix            string = "e2e-cri-uid"
+	defaultNamespacePrefix      string = "e2e-cri-namespace"
+	defaultAttempt              uint32 = 2
+	defaultContainerImage       string = "gcr.io/google_containers/busybox:1.24"
```

Comment:
`dockershim` will pull the image if it does not exist. Is this a `dockershim` specific behavior? And I think runtime should pull the image if it does not exist. It's convenient for users when they create a container which image does not exist. What do you think?